package/libpiksi/libpiksi/src/util.c

```diff
@@ -667,6 +667,56 @@ int run_with_stdin_file2(const char *input_file,
   return status;
 }
 
+bool str_digits_only(const char *str)
+{
+  if (str == NULL) {
+    return false;
+  }
+
+  /* Empty string */
+  if (!(*str)) {
+    return false;
+  }
+
+  while (*str) {
+    if (isdigit(*str++) == 0) {
+      return false;
+    }
+  }
+
+  return true;
+}
+
+bool strtoul_all(int base, const char *str, unsigned long *value)
```

Comment:
Could try to find a common naming scheme for the `str` functions. Now for example the start is `str_` or `str`.

Reply:
I think I'm ok with the inconsistency here, the `strto*_*` functions are meant to be wrappers of the associated library functions, the `str_*` functions are not wrapping anything.